Makefile

```diff
@@ -4,11 +4,11 @@ IMAGE_REGISTRY ?= quay.io
 IMAGE_REPO ?= nmstate
 NAMESPACE ?= nmstate
 
-NMSTATE_CURRENT_COPR_REPO=nmstate-0.3
-NM_CURRENT_COPR_REPO=NetworkManager-1.26
+NMSTATE_CURRENT_COPR_REPO=nmstate-1.0
+NM_CURRENT_COPR_REPO=NetworkManager-1.30
 
-NMSTATE_FUTURE_COPR_REPO=nmstate
-NM_FUTURE_COPR_REPO=NetworkManager-1.26
+NMSTATE_FUTURE_COPR_REPO=nmstate-git
+NM_FUTURE_COPR_REPO=NetworkManager-main
```

Comment:
In the nmstate case, I think this tracks master, including non-released versions. Are you sure you want to depend on non-released code?

Reply:
Right now the "nmstate" copr repo points is the same at "nmstate-1.0" so our only way to test on future changes is with "nmstate-git", let's try this, if we see it breaks too often we go back "nmstate".